core/templates/pages/creator-dashboard-page/creator-dashboard-page.component.html

```diff
@@ -481,13 +482,16 @@ <h2 class="activity-title e2e-test-exp-summary-tile-title">
   }
 
   .oppia-creator-dashboard-main-content .stats-card {
-    min-height: 63px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    min-height: 74px;
     position: relative;
   }
   .oppia-creator-dashboard-main-content .stats-card .stat-description {
-    line-height: 1.646;
-    margin-bottom: 3em;
+    margin-bottom: 0;
     padding-right: 10px;
+    word-wrap: break-word;
```

Comment:
Can you explain why this is needed? Normally we try not to break words.

Reply:
@AFZL210 Thanks for the explanation. Can you show what it looks like with and without this change?

Also, in Arabic is it usual to break words? I would assume not and that line breaks would happen between words, but happy to be corrected on this.

Reply:
OK, I dug into this and I think break-word is probably fine after playing with the examples [here](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break). Thanks!